mobile/app/src/main/java/org/exarhteam/iitc_mobile/IITC_WebViewClient.java

```diff
@@ -259,8 +259,12 @@ public WebResourceResponse shouldInterceptRequest(final WebView view, final Stri
 
         final Uri uri = Uri.parse(url);
         if (uri.getHost() != null && uri.getHost().endsWith(DOMAIN) &&
-                ("http".equals(uri.getScheme()) || "https".equals(uri.getScheme())))
+                ("http".equals(uri.getScheme()) || "https".equals(uri.getScheme()))
+        )
+
+
             return mIitc.getFileManager().getResponse(uri);
+        Log.i("File --> + " +uri);
```

Comment:
The line that's not needed

Reply:
For testing i need it.. because i cannot write to the old path to store the external user plugins.. 

Will also add a TODO here for removing the LOG